wolfssl/wolfcrypt/wc_port.h

```diff
@@ -478,6 +478,12 @@ WOLFSSL_API int wolfCrypt_Cleanup(void);
     #endif
 #endif
 
+#ifdef SIZEOF_TIME_T
+    #if SIZEOF_TIME_T < 8
+        #undef  TIME_T_NOT_LONG
```

Comment:
We should comment that we're testing 64bits, not long.  And potentially change the name.